packages/gatsby-plugin-manifest/src/gatsby-node.js

```diff
@@ -40,8 +40,8 @@ exports.onPostBuild = (args, pluginOptions) =>
       fs.mkdirSync(iconPath)
     }
 
-    fs.writeFileSync(`${iconPath}/manifest.json`, JSON.stringify(manifest))
-    
+    fs.writeFileSync(`./public/manifest.json`, JSON.stringify(manifest))
```

Comment:
@moonmeister what was `iconPath` supposed to be for? It looks like everything is in the root of the public folder anyways?